recipe/meta.yaml

```
@@ -1,42 +1,70 @@
 {% set name = "pytest-celery" %}
-{% set version = "0.0.0a1" %}
+{% set version = "1.2.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest-celery-{{ version }}.tar.gz
-  sha256: 3e0e0817c2d3f2870dafebd915bf13100fc12920b5d42dfe5fdc35844fe42e62
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pytest_celery-{{ version }}.tar.gz
+  sha256: 7873fb3cf4fbfe9b0dd15d359bdb8bbab4a41c7e48f5b0adb7d36138d3704d52
 
 build:
-  number: 1
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [py<=3.8]
```

This will skip python 3.8 too. it is supported according this line: https://github.com/celery/pytest-celery/blob/4ce9278e22675f4e26a96349f4373bfed8336c7c/pyproject.toml#L78
```suggestion
  skip: true  # [py<3.8]
```